src/components/PrivacyPolicy.tsx

```diff
@@ -0,0 +1,111 @@
+import { useState } from 'react';
+import { BottomSheet } from 'react-spring-bottom-sheet';
+import { v4 as uuidv4 } from 'uuid';
+import {
+  PRIVACY_INTRO,
+  privacySections,
+  privacySections2,
+  privacySections3,
+  Table1,
+  Table2,
+} from '@/constants';
+import { PrivacySectionProps } from '@/interfaces';
+import 'react-spring-bottom-sheet/dist/style.css';
+
+const formattedContent = (text: string) => {
+  const LineBreaks = text
+    .replace(/([*|-])\s/g, ' $1 ')
+    .replace(/(\d+\.\s|[\u2460-\u2469]\s)/g, '<br>$1')
+    .split('<br>')
+    .map((item) => {
+      const trimmedItem = item.trim();
+      let paddingLeft = '0px';
+
+      if (trimmedItem.startsWith('*')) {
+        paddingLeft = '16px';
+      } else if (trimmedItem.startsWith('-')) {
+        paddingLeft = '10px';
+      } else if (/^\d+\./.test(trimmedItem)) {
+        paddingLeft = '6px';
+      }
+      return (
+        <div key={uuidv4()} style={{ paddingLeft }}>
+          {item}
+        </div>
+      );
+    });
+  return LineBreaks;
```

Comment:
이 부분은 하나로 통합하는 것이 좋을 것 같아요
```suggestion
return text.replace()...
```

Reply:
수정했습니다.